### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by crookse**
* Makes Documentation section a bulleted list
* Features now match what's on the website with extra nested bullets showing what kinds of body handling Drash can do
* Fixes spacing
* Changes Benchmarks section to direct user to denosaurs/bench repository

### CODE REVIEW

**Review by ebebbington — DISMISSED** [review]
Nice one

File: README.md [hunk 1]
```diff
@@ -165,7 +159,7 @@ Drash does not force you to use all of its code. You can pick and choose which d
 
 ## Benchmarks
 
-Check out our benchmarks [here](https://github.com/drashland/benchmarks).
+Drash's performance is included in the [denosaurs/bench](https://github.com/denosaurs/bench) repository.
```
Comment (saragee3) [inline README.md:162]:
```suggestion
Drash performance benchmarks are included in the [denosaurs/bench](https://github.com/denosaurs/bench) repository.
```

Comment (saragee3) [reply README.md:162]: minor

**Review by saragee3 — APPROVED** [review]
minor suggestion but overall lgtm

**Review by ebebbington — APPROVED** [review]

**Comment by ebebbington** [comment]
Sorry didn't realise

**Pull request merged**